nupic/frameworks/opf/clamodel.py

```diff
@@ -418,7 +420,9 @@ def run(self, inputRecord):
     if self.__logger.isEnabledFor(logging.DEBUG):
       self.__logger.debug("inputRecord: %r, results: %r" % (inputRecord,
                                                             results))
-
+    results.iterationTime = time.clock() - startTime
+    # add to input record (passed to metrics)
+    inputRecord['time'] = results.iterationTime
```

Comment:
@scottpurdy I can do it 2 ways here: 
* append the `time` field to `InputRecord` which is sent to the Metric
* use the functionality of `GlobalStore.get()` as in AnomalyMetric to access TP directly from the metric and read it from there. 

Which one do you think is better? To me it's the former I think.